storm-client/test/jvm/org/apache/storm/utils/UtilsTest.java

```diff
@@ -173,4 +176,23 @@ public void isZkAuthenticationConfiguredStormServerWithPropertyTest() {
             }
         }
     }
+
+    @Test
+    public void testMapDiff() {
```

Comment:
Nitpick: Consider splitting this into multiple small tests each checking one thing